config/default/azwi.yaml

```diff
@@ -0,0 +1,253 @@
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-role
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - admissionregistration.k8s.io
+    resources:
+      - mutatingwebhookconfigurations
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-rolebinding
+  namespace: system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: azure-wi-webhook-manager-role
+subjects:
+  - kind: ServiceAccount
+    name: azure-wi-webhook-admin
+    namespace: system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: azure-wi-webhook-manager-role
+subjects:
+  - kind: ServiceAccount
+    name: azure-wi-webhook-admin
+    namespace: system
+---
+apiVersion: v1
+data:
+  AZURE_ENVIRONMENT: AzurePublicCloud
+  AZURE_TENANT_ID: b39138ca-3cee-4b4a-a4d6-cd83d9dd62f0
+kind: ConfigMap
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-config
+  namespace: system
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-server-cert
+  namespace: system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-webhook-service
+  namespace: system
+spec:
+  ports:
+    - port: 443
+      targetPort: 9443
+  selector:
+    azure-workload-identity.io/system: "true"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-controller-manager
+  namespace: capz-system
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      azure-workload-identity.io/system: "true"
+  template:
+    metadata:
+      labels:
+        azure-workload-identity.io/system: "true"
+    spec:
+      containers:
+        - args:
+            - --arc-cluster=false
+          command:
+            - /manager
+          env:
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.namespace
+          envFrom:
+            - configMapRef:
+                name: azure-wi-webhook-config
+          image: mcr.microsoft.com/oss/azure/workload-identity/webhook:v0.15.0
```

Comment:
There have been changes to the yaml (`Role`, `ClusterRole` and some more). I would recommend updating this to the `v1.0.0` deploy yaml.